src/workers/data-verification.ts

```diff
@@ -63,6 +67,14 @@ export class DataVerificationWorker {
       workerCount,
       streamTimeout,
     });
+
+    this.dataImporter =
+      dataImporter ||
+      new DataImporter({
```

Comment:
This should be instantiated in `system.ts` instead of here. It also needs to use a chunk based `contiguousDataSource` instead of the default background contiguous data source.

Reply:
done